include/klee/Internal/Module/InstructionInfoTable.h

```diff
@@ -50,13 +56,15 @@ namespace klee {
 
     std::string dummyString;
     InstructionInfo dummyInfo;
-    std::map<const llvm::Instruction*, InstructionInfo> infos;
+    unordered_map<const llvm::Instruction *, InstructionInfo> infos;
```

Comment:
add a `std::` in front of it and the following line.
This should fix your compiler issues